serve/mlc_serve/model/paged_cache_manager.py

```diff
@@ -389,9 +390,17 @@ def get_max_new_tokens(self) -> int:
         remaining_blocks = len(self.free_blocks) - free_blocks_per_sequence * len(
             self.allocated_decode_tokens
         )
+
+        total_tokens = []
+
+        for seq_id, tokens in self.allocated_decode_tokens.items():
+            prompt_seq_id = get_prompt_sequence_id(seq_id.request_id)
+            prompt_tokens = self.allocated_prompt_tokens[prompt_seq_id]
+            total_tokens.append(prompt_tokens + tokens)
```

Comment:
This code runs on a hot path and this naive loop incurs noticeable perf regression (5.89 -> 5.78 req / sec) for 13B. However, I suggest merging this as is and follow-up with a non-regressing solution later.